apps/docs/src/app/core/component-docs/shellbar/examples/shellbar-basic-example.component.html

```diff
@@ -1,6 +1,6 @@
 <fd-shellbar>
     <fd-shellbar-logo>
-        <a href="#" class="fd-shellbar__logo fd-shellbar__logo--image-replaced" aria-label="SAP"></a>
+        <a href="#" class="fd-shellbar__logo fd-shellbar__logo--image-replaced" alt="SAP Logo"></a>
```

Comment:
The alt attribute is not allowed for anchor elements: https://stackoverflow.com/questions/14850187/is-it-correct-to-use-alt-tag-for-an-anchor-link/